plugins/modules/monitoring/newrelic_deployment.py

```diff
@@ -52,16 +47,6 @@
     description:
       - The name of the user/process that triggered this deployment
     required: false
-  appname:
-    type: str
-    description:
-      - Name of the application
-    required: false
-  environment:
-    type: str
-    description:
-      - The environment for this deployment
-    required: false
```

Comment:
These also have to stay. If they are not supported by the API anymore, they should be deprecated (and eventually removed).